src/libraries/Common/tests/System/IO/Compression/CompressionStreamUnitTestBase.cs

```diff
@@ -241,16 +241,6 @@ public async Task Read_BaseStreamSlowly()
             }
         }
 
-        [Theory]
-        [InlineData(CompressionMode.Compress)]
-        [InlineData(CompressionMode.Decompress)]
-        public void CanDisposeBaseStream(CompressionMode mode)
-        {
-            var ms = new MemoryStream();
-            using var compressor = CreateStream(ms, mode);
-            ms.Dispose(); // This would throw if this was invalid
-        }
```

Comment:
This test was bogus.  It wasn't disposing of the compression stream, which meant it was actually testing the impact of disposing the base stream.  Changing it to dispose of the compression stream highlights that it does in fact fail with an ObjectDisposedException when trying to flush the compression stream as part of disposal (for BrotliStream it always fails because of the footer, and for the zlib-based streams it fails if there's any data to be flushed).